api/routes/websockets/subscribe.go

```diff
@@ -17,21 +16,20 @@ var (
 		ReadBufferSize:  1024,
 		WriteBufferSize: 1024,
 	}
-	subscribedTopics, _ = NewTopicsFromSlice([]string{"batch", "batch_size", "extract", "transform", "load"})
+	subscribedTopics, _ = newTopicsFromSlice([]string{"^batch.*", "^extract.*", "^transform.*", "^load.*"})
```

Comment:
why doesn't this come from the api/topics package ?

Reply:
oh because it's legacy code from me 😅 but now that you introduced the `topics` package maybe they could be defined there ?